<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WdgGmic</class>
 <widget class="QWidget" name="WdgGmic">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>610</width>
    <height>478</height>
   </rect>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QGridLayout" name="m_filterConfigLayout" columnstretch="1,0,0,0">
     <item row="0" column="0" rowspan="3">
      <widget class="KisGmicInputOutputWidget" name="m_inputOutputOptions"/>
     </item>
     <item row="0" column="1">
      <widget class="QTreeView" name="m_filterTree">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QPushButton" name="updateBtn">
         <property name="text">
          <string>Update definitions</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="expandCollapseBtn">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="2" column="1">
      <layout class="QHBoxLayout" name="horizontalLayout1">
       <item>
        <widget class="QLabel" name="searchLbl">
         <property name="text">
          <string>&amp;Search:</string>
         </property>
         <property name="buddy">
          <cstring>searchBox</cstring>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="searchBox"/>
       </item>
      </layout>
     </item>
     <item row="0" column="2" colspan="2">
      <widget class="QScrollArea" name="m_filterScrollArea">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
         <horstretch>1</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>16777215</height>
        </size>
       </property>
      </widget>
     </item>
     <item row="1" column="2" colspan="2">
      <widget class="QDialogButtonBox" name="controlButtonBox">
       <property name="standardButtons">
        <set>QDialogButtonBox::Apply|QDialogButtonBox::Cancel|QDialogButtonBox::Ok|QDialogButtonBox::Reset</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>KisGmicInputOutputWidget</class>
   <extends>QWidget</extends>
   <header>kis_gmic_input_output_widget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
